docs/tutorials/intro.md

```diff
@@ -70,102 +72,72 @@ algokit init
 
 This will launch a guided menu system to create a specific project tailored to your needs. The templates are basic starter applications for various Algorand development scenarios. To read more about templates checkout AlgoKit detailed documentation.
 
-For now we'll use the `playground` template, which is a lightweight starting point for learning and experimentation. You can initialise a project using this template by running:
+For now we'll use the `python` template, which is a lightweight starting point for learning Algorand smart contract development using Algorand Python. You can initialize a project using this template by running:
 
 ```shell
-algokit init -t playground --no-workspace
+algokit init -t python --no-workspace -a preset_name "starter"
```

Comment:
I'm wondering if we should just use the default and init with a workspace, which would shorten the command to `algokit init -t python -a preset_name "starter"`

Reply:
refreshed a screenshot and added a bit more verbose explanation on the structure that user will see with workspace mode by default